src/components/DocExplorer.js

```diff
@@ -56,48 +110,48 @@ export class DocExplorer extends React.Component {
     );
   }
 
-  // TODO: This part could be optionalized
-  _generateStartPage(schema) {
-    var queryType = schema.getQueryType();
-    var mutationType = schema.getMutationType();
-    var directives = schema.getDirectives();
+  _renderTypeValues(type) {
+    var _renderMarkdown = this._renderMarkdown;
```

Comment:
When `_renderMarkdown` is called below, it throws `TypeError: Cannot read property 'marked' of undefined`. Maybe this line should be `this._renderMarkdown.bind(this)`?